### ISSUE TITLE
Version 0.8.1 and 0.8.2 tag inconsistent on git

### ISSUE THREAD

**Issue by peterjc** [PyXRD/PyXRD#26]
Current release is version 0.8.2, https://github.com/PyXRD/PyXRD/blob/master/pyxrd/__version.py

Most prior releases have a git tag with a leading v, e.g. ``v0.7.5`` which is listed first by GitHub.

The recent 0.8.1 and 0.8.2 releases were tagged without the leading ``v``, and therefore sort to the end of the list on GitHub, leaving the current latest release second from bottom.

I suggest re-tagging them as ``v0.8.1`` and ``v0.8.2`` for consistency and to avoid confusion.

**Comment by mathijs-dumon** [comment]
Indeed, I've renamed (added/deleted) the tags & adjusted the releases accordingly.

**Comment by peterjc** [comment]
Thanks 👍
